src/state.cpp

```diff
@@ -247,7 +248,7 @@ State::commit(const bytes& leaf_secret) const
     next._confirmed_transcript_hash,
     next._interim_transcript_hash,
     next._extensions,
-    std::get<CommitData>(pt.content).confirmation,
+    pt.confirmation_tag.value().mac_value,
```

Comment:
just to double check, are we sure that we will always have confirmation_tag here given it is an optional value ?

Reply:
Yep.  It gets set in `ratchet_and_sign` above.